pkg/types/types.go

```diff
@@ -43,6 +43,7 @@ type NetConf struct {
 	DefaultNetworks []string                 `json:"defaultNetworks"`
 	LogFile         string                   `json:"logFile"`
 	LogLevel        string                   `json:"logLevel"`
+	LogToStderr     bool                     `json:"logToStderr,omitempty"`
```

Comment:
Can we change `LogToStderr` type to `*bool` to keep the default behavior?

Following is the detailed reason. 

If we use 'bool' as a type of `LogToStderr` and no `LogToStderr` in CNI config, then `netconf.LogToStderr` become `false` because golang `omitempty` fills `false` as default and multus cannot identify whether the `false` is user specified or automatically filled as `omitempty`. As a result, multus behavior is changed and it is not expected.

If we use `*bool` then `netconf.LogToStderr` is nul if no LogToStderr in CNI config. Then we could handle it (i.e. set `true` if `nil`) and the behavior is kept.

Reply:
@s1061123 Thanks for your comments. 

The existing code already assures to keep the exisiting default behavior (that LogToStderr's default value is true if there is no logToStderr in CNI config). See code in https://github.com/intel/multus-cni/blob/f97f01f380788b89497515536dec42410d0e1124/pkg/types/conf.go#L261, I've also tested it out.